kirin/__init__.py

```
@@ -32,15 +32,8 @@
 import os
 import sys
 
-from flask_migrate import Migrate
-
-from kirin import exceptions
-
-
 from flask import Flask
 from flask_cors import CORS
-import logging.config
```

Don't we risk a complete crash instead of a log if we import it after something that's crashing?
For example, if there is an issue with `KIRIN_CONFIG_FILE`

---

Moved it back to where it was, doesn't seems to trigger the warning.